api/src/orgs/resolver.js

```diff
@@ -7,7 +14,41 @@ export default {
         db,
         args
       )
+    },
+
+    organisation (_, args, { db }) {
+      return getSingleOrganisation(
+        db,
+        args
+      )
+    }
+  },
+
+  Mutation: {
+    updateProfile (_, args, { db }) {
+      return updateProfile(
+        db, args
+      )
+    }
+  },
+
+  Profile: {
+    name (profile) {
+      return profile.name || ''
+    },
+    icon (profile) {
+      return profile.icon || ''
+    },
+    links (profile) {
+      return profile.links || []
+    },
+    editors (profile) {
+      return profile.editors || []
+    },
+    description (profile) {
+      return profile.description || ''
     }
```

Comment:
In terms of defaults in this section, there is no reason to return empty strings for most of these fields since they are marked as nullable in the schema (i.e. no `!`), and as such I think they should return `null` (except for the arrays which are marked as non-nullable).

Reply:
Done in https://github.com/1Hive/apiary/pull/181/commits/fe3ae6f8174afbe2d00ddbb555a6cb85eacf3228.